CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ Important changes to data-models, configuration and migrations between each
 AppEngine version, listed here to ease deployment and troubleshooting.
 
 ## Next Release (replace with git tag when deployed)
+ * Run `app/bin/tools/backfill_package_fields.dart`.
```

Comment:
Why this? Have we changed anything in how these are derived?

Reply:
We've added `latestPublished` and `latestPrereleasePublished` in the previous release, we'll need to backfill them in the current one. It is slightly related to this change (eventually we want to use these as updated dates), that's the reason I've added it.